data-raw/rs_onco_irecist.R

```diff
@@ -0,0 +1,388 @@
+# Packages: ---------------------------------------------------------------
+library(datapasta)
+library(dplyr)
+
+# Tribble data: -----------------------------------------------------------
+rs_onco_irecist <- tibble::tribble(
+       ~USUBJID, ~RSSEQ, ~RSLNKGRP,  ~RSTESTCD,                             ~RSTEST,           ~RSORRES,          ~RSSTRESC,    ~RSSTAT,                        ~RSREASND, ~VISITNUM,            ~VISIT,       ~RSDTC, ~RSDY,
+  "01-701-1015",     7L,      "A2", "OVRLRESP",                  "Overall Response",             "iUPD",             "iUPD",         NA,                               NA,         7,          "WEEK 6", "2014-02-12",   42L,

```

Comment:
```suggestion
  ~USUBJID,      ~RSSEQ, ~RSLNKGRP,  ~RSTESTCD,                             ~RSTEST,           ~RSORRES,          ~RSSTRESC,    ~RSSTAT,                        ~RSREASND, ~VISITNUM,            ~VISIT,       ~RSDTC, ~RSDY,
  "01-701-1015",     7L,      "A2", "OVRLRESP",                  "Overall Response",             "iUPD",             "iUPD",         NA,                               NA,         7,          "WEEK 6", "2014-02-12",   42L,

```
The first column needs to be left aligned. Otherwise, `styler` removes the alignment.